packages/react-native/React/Views/UIView+React.m

```diff
@@ -205,22 +206,23 @@ - (void)reactSetFrame:(CGRect)frame
   self.bounds = bounds;
 
   id transformOrigin = objc_getAssociatedObject(self, @selector(reactTransformOrigin));
-  if (transformOrigin) {
+  id transform = objc_getAssociatedObject(self, @selector(reactTransform));
+  if (transformOrigin || transform) {
     updateTransform(self);
   }
```

Comment:
There's going to be a performance cost to parsing this more frequently (I guess anytime the frame changes right now?). Could we limit the updates to only when the bounds have changed?

Reply:
Thanks for catching, updated.